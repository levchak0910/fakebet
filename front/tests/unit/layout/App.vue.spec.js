import {createWrapper, disableFile} from "../__utils__";
import Component from "@/App.vue";

disableFile();

describe("layout/App.vue", () => {

    let wrapper;
    let mountWrapper = (options = {}) => {
        wrapper = createWrapper(Component, {
            stubs: ["app-header", "router-view"],
            ...options
        });
    };

    it("Testing snapshot", () => {
        mountWrapper();

        expect(wrapper.element).toMatchSnapshot();
    });

});
