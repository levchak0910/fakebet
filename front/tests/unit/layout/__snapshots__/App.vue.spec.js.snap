// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout/App.vue Testing snapshot 1`] = `
<div
  id="app"
>
  <app-header-stub />
  <router-view-stub />
</div>
`;
